bin/fix-psp-violations.bash

```diff
@@ -7,6 +7,11 @@ usage() {
 
 ROOT="$(readlink -f "$(dirname "${0}")/../")"
 
+export CK8S_CLUSTER="${1}"
```

Comment:
With the changes of [this PR here](https://github.com/elastisys/compliantkubernetes-apps/pull/1778/files#diff-d157d0f7a32fe5a99591a42778e449309300204efe637caef726eefdceb9f0deR181-R186), since the `fix-psp-violations.bash` script sources the `lib.sh` script which runs `check_config`, `CK8S_CLUSTER` needs to be set, otherwise you get this:
```console
✗ ./bin/ck8s fix-psp-violations wc  
[ck8s] fix-psp-violations.bash: error: "CK8S_CLUSTER" is unset
```

But I agree moving it one level up for all commands might be good.